### ISSUE TITLE
Grammatical error

### ISSUE THREAD

**Issue by ryanauj** [aspnet/AspNetCore.Docs#16471]
Very small grammatical error in the bullet point:
```
- IdentityServer with an additional AddApiAuthorization helper method that setups some default ASP.NET Core conventions on top of IdentityServer
```
I would like to propose a change from 'setups' to 'sets up'. This was bugging me while reading it, and even though it is a small change I think it makes it a bit more readable.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 21cb7455-3305-9316-5440-bdc22947a82d
* Version Independent ID: 137d4b94-7b26-3911-b22d-42c754a95fc1
* Content: [Introduction to authentication for Single Page Apps on ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity-api-authorization?view=aspnetcore-3.1)
* Content Source: [aspnetcore/security/authentication/identity-api-authorization.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/authentication/identity-api-authorization.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @javiercn
* Microsoft Alias: **scaddie**

**Comment by guardrex** [comment]
Thanks @ryanauj ... I'll patch that now.

**Comment by guardrex** [comment]
Oh ... @ryanauj ... you didn't put a `Fixes #16471` in the opening comment of your PR; therefore, this issue didn't close automatically when the PR merged.
